lambdas/test-lambda/build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+    id "java"
+    id "jacoco"
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation "com.amazonaws:aws-lambda-java-core:$rootProject.ext.dependencyVersions.awsLambdaJavaCore"
+
+    testImplementation "org.junit.jupiter:junit-jupiter:5.9.3",
```

Comment:
I think I said this on a different review somewhere, but if we're pinning dependency versions anywhere (e.g. line 11) can we pin them the same way everywhere?

Reply:
Done